tower-timeout/Cargo.toml

```diff
@@ -7,4 +7,8 @@ publish = false
 [dependencies]
 futures = "0.1"
 tower-service = { version = "0.2", path = "../tower-service" }
-tokio-timer = "0.2.6"
```

Comment:
Unless there's a specific reason for this change, my understanding is that it's best for libraries to depend on the component crates, rather than `tokio`, especially since the 0.2.x branch of `tokio-timer` is _not_ deprecated.

Reply:
Ah, I confused https://github.com/tokio-rs/tokio/blob/master/tokio-timer/ with https://github.com/tokio-rs/tokio-timer. Sorry about that. I'll revert the changes.